master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -88,6 +90,43 @@ func GetTaskConfigPolicies(ctx context.Context,
 	return &tcp, nil
 }
 
+// GetPriorityLimit reads the priority limit for the given scope and workload type.
+// It returns found=false if no limit exists.
+func GetPriorityLimit(ctx context.Context, scope *int, workloadType string) (limit int, found bool, err error) {
+	if !ValidWorkloadType(workloadType) {
+		return 0, false, fmt.Errorf("invalid workload type: %s", workloadType)
```

Comment:
Should we add an empty string check in here as well?
With the empty string, the error message shows up as `invalid workload type: `. I like specifically checking for the empty string so that we can return a cleaner error message!

Reply:
In this case, I don't think it's worth the extra lines of code because workload type is not user input - it's hard-coded elsewhere in our code. Generally, that's a great idea though!

Reply:
I gotchaa, makes sense thanks for explaining!